src/Compilers/VisualBasic/Portable/Symbols/Source/SynthesizedMyGroupCollectionPropertyAccessorSymbol.vb

```diff
@@ -169,10 +170,12 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
             '    return <backingField>
             ' End Get
 
-            Return "Get" & vbCrLf &
```

Comment:
@gafter  String Literals have significant whitespace within their double quotes. If the multi-line string literal is indented, indentation is included within the string. Highly likely not what the user intended. See #2231 I think a comment explaining break indentation.